engine/buildkit/containerfs/fs.go

```diff
@@ -282,12 +282,18 @@ func (ctrFS *ContainerFS) PathExists(path string) (bool, error) {
 
 func (ctrFS *ContainerFS) OSReleaseFileContains(ids [][]byte, idLikes [][]byte) (bool, error) {
 	f, err := ctrFS.Open("/etc/os-release")
-	if err != nil {
-		if errors.Is(err, os.ErrNotExist) {
-			return false, nil
-		}
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
nit the err != nil can be omitted here and below since errors.Is already checks for that